docs/source/host/configure-settings/place-setting.rst

```diff
@@ -14,18 +10,15 @@ Place names, on the other hand, make it easy to travel to your domain. They're m
     :depth: 2
 
 -----------------------------
-Purchase a Place Name
+Get a Place Name
 -----------------------------
 
-Place names are purchased from Vircadia for $20/year. Once purchased, a place name can be transferred freely between any of your domains.
-
-.. note:: If you would like to use HFC to purchase a place name, `schedule an appointment with the Vircadia banker <https://meetings.hubspot.com/highfidelity/high-fidelity-bank-appointment>`_. Here, you can purchase a coupon that can be used to purchase a place name using the process below.
+Once acquired, a place name can be transferred freely between any of your domains.
 
-1. Go to https://highfidelity.com/user/places to purchase a place name. If you’re not logged in, you will be prompted to do so.
-2. Click 'Create a Place Name'.
-3. Type in your new Place Name. Place Names must be 4-64 characters in length. Letters, numbers and hyphens are allowed. You can use any place name that has not been already purchases; however, please note that Vircadia reserves the right to reject any registration we deem obscene, offensive, or a copyright violation, and refund the corresponding registration fees.
-4. Click 'Checkout Now'.
-5. Enter your credit card number, PayPal account information or a coupon code to complete the purchase.
+1. Go to https://dashboard.vircadia.com/places to get a place name. If you're not logged in, you will be prompted to do so.
+2. Click 'Create Place'.
+3. Type in your new Place Name. Place Names must be 4-64 characters in length. Letters, numbers and hyphens are allowed. You can use any place name that has not already been taken; however, please note that Vircadia reserves the right to reject any registration we deem obscene, offensive, or a copyright violation, and refund the corresponding registration fees.
```

Comment:
This information doesn't seem valid anymore. The metaverse server documentation doesn't talk about any limits https://github.com/vircadia/Iamus/blob/master/docs/API-Places.md#post-apiv1userplaces

Actually I just created a placename with 128 characters using a password generator: `MQrzpZTLJSGWBSM4yvP4X4$3iM&5Rwzg!7k#u*CmUFyNHwcv@@oM#Ngp#@gx5gfVEoNWbzMP3%5huL#$a$s5ReXw7j$izhXBX@jcq5tn@uPWN9XhYnSLA9H74%H6U4bw`

Reply:
understood, i'll remove the part about length requirements.

Reply:
i'll also remove the thing about letters numbers and hyphens since clearly a lot of other things also work